docs/src/aleph_bft_api.md

```diff
@@ -22,7 +22,7 @@ pub trait FinalizationHandler<Data> {
 }
 ```
 
-Calls to function `data_finalized` represent the order of the units that AlephBft produced.
+Calls to function `data_finalized` represent the order of the units that AlephBFT produced and that hold `Some(data)`.
```

Comment:
s/`Some(data)`/some data

Documentation is for humans. :P

Reply:
controversial statement